miden-lib/asm/faucets/basic.masm

```diff
@@ -0,0 +1,91 @@
+# BASIC FUNGIBLE FAUCET CONTRACT
+# =================================================================================================
+# This is a basic fungible faucet smart contract. 
+#
+# It allows the owner of the faucet to mint, distribute, and burn tokens. Token metadata is stored 
+# at position 1 as [token_symbol, decimals, 0, max_supply], where:
```

Comment:
We should make the comment more explicit about where the metadata is stored. I'd change the second sentence to something like: "Token metadata is stored in account storage in slot 1 as ...".

Reply:
done